.github/workflows/ci.yml

```diff
@@ -12,8 +12,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        llvm: ["800", "900", "1000"]
-        ubuntu: ["20.04", "19.10", "18.04"]
+        llvm: ["900", "1000"]
```

Comment:
Let's add LLVM8 back in once we make Anvill run on LLVM8; i don't want to abandon LLVM8 support just yet, but will be fine letting it go if it becomes too much of a headache to maintain.